src/js/midi/ui.js

```diff
@@ -0,0 +1,80 @@
+const MidiChannel = ({ type, name, id, enabled }) => {
+  const template = document.createElement('template')
+  template.innerHTML = `
+    <div style="display:flex;flex-direction:column;padding:0 10px;align-items:center">
```

Comment:
Is this inline styling temporary? Should be moved to style sheets.

Reply:
Yes, it is temporary, I'm just being the lazy :D